docs/calico-with-docker/docker-network-plugin/vagrant-ubuntu/Vagrantfile

```diff
@@ -69,9 +65,6 @@ Vagrant.configure(2) do |config|
       # Set Docker to use etcd for multihost.
       host.vm.provision :shell, inline: %Q|sudo sh -c 'echo "DOCKER_OPTS=\"--cluster-store=etcd://#{primary_ip}:2379\"" > /etc/default/docker'|
 
```

Comment:
We are currently setting the cluster store using the docker daemon command line options.  If we do that, don't we need to restart the docker daemon to pick this up?  I would have not expected this to work - does it?

Perhaps we should also change to use the daemon.json file and then just send in a SIGHUP